internnav/habitat_extensions/evaluator.py

```diff
@@ -623,10 +624,39 @@ def eval_action(self, idx) -> None:
                             action = action.detach().cpu().numpy()[0] if isinstance(action, torch.Tensor) else action
                             action = action[0] if hasattr(action, "__len__") else action
                         else: # dual-system logic
-                            if len(local_actions) != 0:
+                            if len(local_actions)==0:
+                                ############nav_dp###########
+                                local_actions = []
+                                image_dp = torch.tensor(np.array(look_down_image.resize((224, 224)))).to(torch.bfloat16) / 255
```

Comment:
Generally, we do not use hard code. It's better to change it into a parameter, or leave the comments to explain why to resize it into 224.